specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2021-01-01-preview/namespace-preview.json

```diff
@@ -759,32 +764,59 @@
       "properties": {
         "principalId": {
           "type": "string",
-          "description": "ObjectId from the KeyVault"
+          "description": "ObjectId from the KeyVault",
+          "readOnly": true
         },
         "tenantId": {
           "type": "string",
-          "description": "TenantId from the KeyVault"
+          "description": "TenantId from the KeyVault",
+          "readOnly": true
         },
         "type": {
-          "type": "string",
-          "description": "Enumerates the possible value Identity type, which currently supports only 'SystemAssigned'",
-          "default": "SystemAssigned",
+          "description": "Type of managed service identity.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
+            "None"
           ],
+          "type": "string",
           "x-ms-enum": {
-            "name": "IdentityType",
+            "name": "ManagedServiceIdentityType",
             "modelAsString": false
           }
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
```

Comment:
This is an anonymous type. Please provide a "x-ms-client-name", or define it in "#definitions".